bundles/org.openhab.core.io.rest.sitemap/src/main/java/org/openhab/core/io/rest/sitemap/internal/MappingDTO.java

```diff
@@ -17,10 +17,12 @@
  *
  * @author Kai Kreuzer - Initial contribution
  * @author Laurent Garnier - New fields position and icon
+ * @author Laurent Garnier - Replace field position by fields line and column
  */
 public class MappingDTO {
 
-    public Integer position;
+    public Integer line;
```

Comment:
Could you call it row instead of line?
In the UI builder, I would put placeholders in the fields 'row' and 'col' where a number has to go. 'line' is longer, and I think row is the better term. I can use different terminology in the UI sitemap builder, but it is clearer keeping it consistent.

Reply:
Ok, changed